init.lua

```diff
@@ -185,3 +190,4 @@ toolranks.add_tool("default:shovel_steel")
 toolranks.add_tool("default:shovel_bronze")
 toolranks.add_tool("default:shovel_mese")
 toolranks.add_tool("default:shovel_diamond")
+
```

Comment:
Looks like there are two endlines here already. GitHub doesn't show the endline, though.
```suggestion
```